File: LV1/폰켓몬.js
function solution(nums) {
  let answer = 0;
  let canSelect = nums.length / 2;
  const set = new Set(nums);
  const uniqueArr = [...set];
  let poketmon = uniqueArr.length;
  if (canSelect == poketmon) {
    answer = poketmon;
  } else if (canSelect > poketmon) {
    answer = poketmon;
  } else {
    answer = canSelect;
  }
  return answer;
}

File: LV1/폰켓몬.py
def solution(nums):
    answer = 0
    canSelect = len(nums) / 2
    uniqueArr = list(set(nums))
    poketmon = len(uniqueArr)
    if canSelect == poketmon:
        answer = poketmon
    elif canSelect > poketmon:
        answer = poketmon
    else:
        answer = canSelect
    return answer
